qa/L0_client_memory_growth/test.sh

```diff
@@ -117,6 +119,7 @@ for PROTOCOL in http grpc; do
             fi
 
             # Log the graph for memory growth and the change between Average and Max memory usage
```

Comment:
Can you update the above comment to include test duration?

Reply:
Updated